docs/image-redactor/index.md

```diff
@@ -1,22 +1,113 @@
 # Presidio Image Redactor
 
+***Please notice, this package is still in beta and not production ready.***
+
 ## Description
 
-## Table of contents
+The Presidio Image Redactor is a Python based module for detecting and redacting PII
+text entities in images.
+![img.png](../assets/image-redactor-design.png)
 
 ## Installation
 
-## Getting started
+Pre-requisites:
+
+- Install [Tesseract OCR](https://github.com/tesseract-ocr/tesseract#installing-tesseract) by following the
+  instructions on how to install it for your operating system.
+
+  The Image-Redactor was tested with a minimal version of 4.0.0
 
-- Running in Python.
-- Running using Docker.
-- Customizing.
-- API reference.
+=== "Using pip"
+    
+    !!! note "Note"
+        Consider installing the Presidio python packages on a virtual environment like venv or conda.
+    
+    To get started with Presidio-image-redactor,
+    download the package and the `en_core_web_lg` spaCy model:
+    
+    ```sh
+    pip install presidio-image-redactor
+    python -m spacy download en_core_web_lg
+    ```
 
-### Running in Python
+=== "Using Docker"
```

Comment:
nip: consider using standard headers

Reply:
WDYM by standard header? 
This is part of a menu.

Reply:
![Screen Shot 2021-03-01 at 12 27 07](https://user-images.githubusercontent.com/1862328/109484731-752e2a00-7a89-11eb-8b55-0b3bc5a174f8.png)

Reply:
my bad